deploy-edgeX.sh

```diff
@@ -89,6 +89,8 @@ run_service device-virtual
 
 run_service app-service-configurable
 
+run_service system
```

Comment:
For consistency, can we move this up to be near the other `edgex-go` services? This goes for the logging Docker `log` command at the bottom of this file as well.